requirements_test.txt

```diff
@@ -16,3 +16,4 @@ pytest-sugar==0.9.2
 pytest-timeout==1.3.3
 pytest==4.4.0
 requests_mock==1.5.2
+aioswitcher==2019.3.21
```

Comment:
Don't add here, this is only for the core testing libs.  Use `script/gen_requirements_all.py` to update the requirements files - do NOT manually edit them.

Reply:
Fixed in this [commit](https://github.com/home-assistant/home-assistant/pull/22325/commits/c18e74222cd98c988bd687fd00994a928c8f1ca8).